tests/testthat/test-misc.R

```diff
@@ -117,3 +117,12 @@ test_that("vectorOrList", {
     expect_true(vectorOrList(c(1, 2, 3), "numeric"))
     expect_false(vectorOrList(list("a", 1, "c"), "numeric"))
 })
+
+test_that("setCrunchAPI", {
+    old_crunch_api <- getOption("crunch.api")
+    setCrunchAPI('foobar')
+    expect_equal(getOption("crunch.api"), "https://foobar.crunch.io/api/")
+    setCrunchAPI('barfoo', 8888)
+    expect_equal(getOption("crunch.api"), "http://barfoo.crunch.io:8888/api/")
+    options("crunch.api" = old_crunch_api)
```

Comment:
It's generally better to do something like this as an `on.exit` up at the top of this block so that you ensure that it is called. Otherwise if the code on L123-126 errors, it will stop execution before resetting your option.

But even better for this case, use the `reset.option` context: https://github.com/Crunch-io/rcrunch/blob/81b02b9636fcce2ec8342ca3aab3529b15ab3044/tests/testthat/helper-contexts.R#L44